package.json

```diff
@@ -1,10 +1,35 @@
 {
+  "version": "2.35.0",
+  "scripts": {
+    "test": "mocha && eslint ."
+  },
+  "license": "MIT",
+  "main": "index.js",
+  "name": "apostrophe-workflow",
   "author": {
     "name": "Apostrophe Technologies LLC"
   },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/apostrophecms/apostrophe-workflow.git"
+  },
+  "homepage": "https://github.com/apostrophecms/apostrophe-workflow#readme",
+  "keywords": [
+    "apostrophe",
+    "apostrophecms",
+    "apostrophe-cms",
+    "content management",
+    "content management system",
+    "cms",
+    "workflow",
+    "localization",
+    "editorial"
+  ],
   "bugs": {
     "url": "https://github.com/apostrophecms/apostrophe-workflow/issues"
   },
+  "deprecated": false,
+  "description": "Workflow, approvals, localization and internationalization for ApostropheCMS",
   "bundleDependencies": false,
   "peerDependencies": {
```

Comment:
Are there two peerDep entries now?

Reply:
No, just the one at the bottom.

Reply:
oh, maybe. I missed that in the merge conflict ui.